lib/Reply.js

```diff
@@ -23,12 +23,13 @@ class Reply {
 
     this.alexaEvent = alexaEvent;
     this.session = alexaEvent.session;
+
     this.msg = {
-      statements: [],
-      reprompt: '',
-      card: undefined,
-      yield: false,
-      hasAnAsk: false,
+      statements: [], // Statements will be concatenated together before being sent
+      reprompt: '', // Since only one reprompt is possible, only the latest value is kept
```

Comment:
@heneryville seems to me we should support building reprompts the same way we build ask or tell statements.

Reply:
I'm not sure that makes sense. A reprompt should normally be a short single question rather than a long statement that could be constructed. A reprompt with two questions in it is confusing. Heck, if there are ever two reprompts it's probably an error, but for flexibility, overwriting the last is also reasonable.

Reply:
My reasoning is that the rendering system should behave similar everywhere, seems to me that while using the system it would feel more natural to expect the reprompt to work the same way as the `say` key